common/src/main/java/com/viaversion/viaversion/protocols/v1_10to1_11/Protocol1_10To1_11.java

```diff
@@ -41,8 +39,11 @@
 import com.viaversion.viaversion.protocols.v1_9_1to1_9_3.packet.ServerboundPackets1_9_3;
 import com.viaversion.viaversion.rewriter.SoundRewriter;
 import com.viaversion.viaversion.util.Pair;
+import com.viaversion.viaversion.util.ProtocolLogger;
 
 public class Protocol1_10To1_11 extends AbstractProtocol<ClientboundPackets1_9_3, ClientboundPackets1_9_3, ServerboundPackets1_9_3, ServerboundPackets1_9_3> {
+
+    public static final ProtocolLogger LOGGER = new ProtocolLogger(Protocol1_10To1_11.class);
```

Comment:
These should be moved to instance fields in AbstractProtocol with a getter there

Reply:
Done, I'll work on using Protocol#getLogger() in rewriter bases later